pageserver/src/tenant/timeline/detach_ancestor.rs

```diff
@@ -1014,3 +1029,13 @@ fn check_no_archived_children_of_ancestor(
     }
     Ok(())
 }
+
+async fn fsync_timeline_dir(path: impl AsRef<Utf8Path>, ctx: &RequestContext) {
```

Comment:
Why can't this take a `&Arc<Timeline>` and figure out the path itself?

Reply:
changed